### PR TITLE
Add i18next and japanese translation

### PR DESCRIPTION
**Pull request by ThibautBremand**

### CODE REVIEW

File: packages/extension/src/App.tsx [hunk 1]
```diff
@@ -94,136 +94,138 @@ const App: FC = () => {
...
-        <Route
-          path={TRUSTED_APPS_PATH}
-          element={
-            <PrivateRoute>
-              <TrustedApps />
-            </PrivateRoute>
-          }
-        />
-      </SentryRoutes>
+      <Suspense fallback="loading">
```
Comment (ThibautBremand) [inline packages/extension/src/App.tsx:97]: @FlorianBouron Here is the Suspense usage for making i18next work

**Comment by FlorianBouron** [comment]
@all-contributors please add @ThibautBremand  for code

File: packages/extension/src/App.tsx (hunk 1 above)
Comment (ThibautBremand) [reply packages/extension/src/App.tsx:97]: @FlorianBouron I can't put <Suspense> within <SentryRoutes>, the app would not render

File: packages/constants/src/network/network.constant.ts [hunk 2]
```diff
@@ -12,26 +12,27 @@ export enum NetworkServer {
...
 }
 
+// Network description only embeds the translation key. You can pass the translation key to i18n.t() to get the
+// translation.
 export const NETWORK = {
   [Network.MAINNET]: {
     name: Network.MAINNET,
     server: NetworkServer.MAINNET,
-    description: 'Main network using the production version of the XRP Ledger.'
+    description_key: 'TEXT_NETWORK_DESCRIPTION_MAINNET'
```
Comment (ThibautBremand) [inline packages/constants/src/network/network.constant.ts:21]: @FlorianBouron FYI

**Pull request closed without merging**